_source/atoms/headline/Headline.js

```diff
@@ -14,6 +14,7 @@ export default class Headline extends Component {
           display && 'headline--display',
           noMargin && 'headline--no-margin',
           color && `headline--color-${color}`,
+          darkMode && 'headline--dark-mode',
```

Comment:
Can we merge `headline--color-${color}` and `headline--dark-mode` here?

Reply:
I'll check!